src/reducers/index.js

```diff
@@ -8,8 +8,10 @@ import { INCREMENT, DECREMENT } from '../actions';
 export default (count = 0, action) => {
   switch (action.type) {
     case INCREMENT:
+      return (count += action.payload);
```

Comment:
Whole point of using Redux is state management and possibility to "remember" the previous state and not modify the current one. In this particular case You should either call Object.assign or use other measures to preserve previous state and instead of modify current one, create a new "copy" of it with modified value.